### PR TITLE
Add desktop version

### PR DESCRIPTION
**Pull request by Drod86**
<h2>In this branch:</h2>

- Changed the flex-direction of the feature project, about, and skills sections
- Adjusted the padding of every section
- Inserted the desktop versions of the main and about section graphics

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -35,6 +35,7 @@ header {
   position: sticky;
   top: 0;
   z-index: 2;
```
Comment (usorfaitheloho) [inline styles.css:37]: You did welll with your portfolio website, I'd kindly recommend that you add some margins to your heasline section as it looks too close to the end of the screen and isnt similar to the given design 😄 

Figma design :
image

Your design :
image

File: styles.css [hunk 2]
```diff
@@ -568,44 +582,147 @@ footer .socials-box {
...
-    border: 1px solid red;
     width: 46%;
     display: inline-block;
   }
 
+  .recent-works {
+    position: relative;
+    padding: 15vw 3.75vw 7.5vw;
+  }
+
```
Comment (usorfaitheloho) [inline styles.css:600]: Kindly add a margin-top to the `My Recent Works ` title and center align it as it is in the figma template 👍 

Your design :
image

Figma design :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @Drod86  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (Drod86) [reply styles.css]: Hello @usorfaitheloho,
I adjusted the margins as you suggested. Thank you for the feedback.

**Review by Efiamotu-1 — APPROVED** [review]
Hi @Drod86, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

giphy


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* Good PR Title & summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**